torch/csrc/deploy/README.md

```diff
@@ -8,3 +8,15 @@ This is an internal library used behind the scenes to enable multiple python int
 a single deploy runtime.  libinterpreter.so is DLOPENed multiple times by the deploy library.
 Each copy of libinterpreter exposes a simple interpreter interface but hides its python and other
 internal symbols, preventing the different python instances from seeing each other.
+
+# cpython build
+Torch Deploy builds cpython from source as part of the embedded python interpreter.  Cpython has a flexible build system that builds successfully with or without a variety of dependencies installed - if missing, the resulting cpython build simply omits optional functionality, meaning some stdlib modules/libs are not present.
+
+Currently, the torch deploy build setup assumes the full cpython build is present.  This matters becuase there is a [hardcoded list of python stdlib modules](https://github.com/pytorch/pytorch/blob/master/torch/csrc/deploy/interpreter/interpreter_impl.cpp#L35) that are explicitly loaded from the embedded binary at runtime.
```

Comment:
because

Reply:
If you quote a line number you should pin the link here to a specific hash so it doesn't drift